man/systemd.network.xml

```diff
@@ -1732,6 +1732,17 @@
           </listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><varname>AssignAcquiredDelegatedPrefix=</varname></term>
```

Comment:
Maybe call this AssignAcquiredDelegatedPrefixAddress=? I mean, this doesn't assign the prefix, but an automatically determined IP address from it, right?

Reply:
yes